spec/system/posts_show_integration_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Show Page', type: :system do
+  before do
+    @user = create(:user)
+    @post = create(:post, author: @user)
+    @comments = create_list(:comment, 3, post: @post, author: @user)
+
+    visit user_post_path(@user, @post)
+  end
+
+  scenario 'displays the post\'s title' do
+    expect(page).to have_content("Post: #{@post.id} by: #{@user.name}")
+  end
+
+  scenario 'displays the post\'s author' do
+    expect(page).to have_content("by: #{@post.author.name}")
+  end
+
+  scenario 'displays the post\'s author' do
+    expect(page).to have_content("by: #{@post.author.name}")
+  end
+
```

Comment:
- [ ] @Team Great work adding tests for the posts show page, however, you have two similar tests, kindly consider removing one of them, it's always good practice to avoid code repetition in the codebase.